tests/test_category_views.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+
+from tests.factories import CategoryFactory, ArticleFactory
+
+
+@pytest.mark.parametrize('num_of_categories', [1, 3, 0])
+def test_categories_can_be_listed(client, num_of_categories):
+    CategoryFactory.create_batch(size=num_of_categories)
+    response = client.get('/categories')
+
+    assert response.status_code == 200
+    assert response.content_type == 'application/json'
+    assert len(response.json['items']) == num_of_categories
+
+
+def test_get_category_by_id(client):
+    article = ArticleFactory()
+    category = CategoryFactory(articles=[article])
+    response = client.get('/categories/%s' % category.id)
+
+    assert response.status_code == 200
+    assert response.content_type == 'application/json'
+    expected = {
+        'id': category.id,
+        'name': category.name,
+        'articles': [
```

Comment:
would omit `articles` here for now, unless it's strictly needed by `jats-ingester`; I understand it's part of the default serialization but you I see no hard and fast rule for omitting or including the various relationships the `Category` is linked to inside the relational database.

Reply:
updated.